docs/guides/postgres/branch/customization/index.md

```
@@ -0,0 +1,532 @@
+description: Customizing a PostgreSQL branch — credentials, post-actions, scheduled refresh, target namespace, HA, snapshot class
+menu:
+  docs_{{ .version }}:
+    identifier: guides-pg-branch-customization
+    name: Customizing Branches
+    parent: guides-pg-branch
+    weight: 30
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)